website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -2,28 +2,114 @@
 
 ### When to use
 
-- When a user’s action does not yield any results due to incorrect or incomplete input.
 - When an application encounters an issue or error during its operation.
+- When an application needs emphasis on the creation of a new object within a null state.
 
 ### When not to use
 
 - When the absence of content is expected and does not require an explanation to the user.
 - When there is a clear and intuitive way to add or populate content.
 
-## Icon
+## Alignment
 
-- The icon should align with the purpose of the content and effectively communicate the same message.
+The Application State supports two alignment options: `left` and `center`. The alignment affects text alignment, action placement/alignment in the footer, and media placement; however, it does not change the default page alignment.
 
-## Actions
+!!! Info
 
-- In the footer, you can include up to two stand-alone links. 
-- We don’t recommend using buttons, as most actions will navigate the user away from this page. Learn more about [when to use a link vs. a button](https://helios.hashicorp.design/components/link/standalone#usage).
-- For the standalone link, we recommend using the medium size.
-- Use footer actions to redirect or guide users in solving errors/access issues with actionable steps.
+By default, the Application State has horizontal auto margins applied to it, always centering it on the page. This can be overridden with CSS properties.
```

Comment:
[question] Is this also true depending on the container the component is used within?

```suggestion
By default, the Application State has horizontal auto margins applied to it, always centering it on the page or containing element. This can be overridden with CSS properties.
```